lettuce/django/management/commands/harvest.py

```diff
@@ -35,110 +34,103 @@
 from lettuce.django.server import LettuceServerException
 
 
+DJANGO_VERSION = StrictVersion(django.get_version())
+
+
 class Command(BaseCommand):
     help = u'Run lettuce tests all along installed apps'
     args = '[PATH to feature file or folder]'
-    requires_model_validation = requires_system_checks = False
-
-    option_list = BaseCommand.option_list + (
-        make_option('-a', '--apps', action='store', dest='apps', default='',
-            help='Run ONLY the django apps that are listed here. Comma separated'),
 
-        make_option('-A', '--avoid-apps', action='store', dest='avoid_apps', default='',
-            help='AVOID running the django apps that are listed here. Comma separated'),
-
-        make_option('-S', '--no-server', action='store_true', dest='no_server', default=False,
-            help="will not run django's builtin HTTP server"),
-
-        make_option('--nothreading', action='store_false', dest='use_threading', default=True,
-            help='Tells Django to NOT use threading.'),
-
-        make_option('-T', '--test-server', action='store_true', dest='test_database',
+    if DJANGO_VERSION < StrictVersion('1.9'):
```

Comment:
I'll still support 1.6 here, and this will need to be removed when support for 1.6 is dropped (I don't think it is). This file still supports 1.6 (see below)